.azure/pipelines/jobs/default-build.yml

```diff
@@ -58,7 +58,7 @@ parameters:
   installTar: true
   installNodeJs: true
   installJdk: true
-  timeoutInMinutes: 270
+  timeoutInMinutes: 180
```

Comment:
?

Reply:
@wtgodbe undid a workaround for some long-running builds